src/iarc7_motion/motion_planner.py

```diff
@@ -48,7 +54,9 @@ def __init__(self, _action_server, update_rate):
             task_commands.VelocityCommand: self._handle_velocity_command,
             task_commands.ArmCommand: self._handle_arm_command,
             task_commands.NopCommand: self._handle_nop_command,
-            task_commands.GroundInteractionCommand: self._handle_ground_interaction_command
+            task_commands.GroundInteractionCommand: self._handle_ground_interaction_command,
+            task_commands.PassthroughMode: self._handle_passthrough_mode_command,
+            task_commands.PassthroughCommand: self._handle_passthrough_command
```

Comment:
AngleThrottleCommand? Really we should have names for the different controllers.